classes/class-woothemes-sensei-certificates.php

```diff
@@ -133,7 +133,7 @@ public static function init() {
 
 		// Filters
 		add_filter( 'sensei_user_course_status_passed', array( $instance, 'certificate_link' ), 10, 1 );
-		add_filter( 'sensei_results_links', array( $instance, 'certificate_link' ), 10, 2 );
+		add_filter( 'sensei_results_completed_links', array( $instance, 'certificate_link' ), 10, 3 );
```

Comment:
It would be ideal if we could keep using the `sensei_results_links` hook because it is triggered in [multiple places](https://github.com/woocommerce/sensei-certificates/assets/1612178/a8aa5353-6045-4956-9c85-404401efae43) in the Sensei LMS codebase. I've left a suggestion in your [Sensei LMS PR](https://github.com/Automattic/sensei/pull/7048#discussion_r1279773144) related to that. I did a quick test using the suggested code and it looks like it's working but let me know if I missed something (totally possible!).

Reply:
Made those changes work on the Sensei LMS and updated this filter in 2ccaa84.

Thanks!